java-client/src/test/java/org/opensearch/client/opensearch/integTest/OpenSearchJavaClientTestCase.java

```diff
@@ -45,7 +45,12 @@
 import org.opensearch.test.rest.OpenSearchRestTestCase;
 
 public abstract class OpenSearchJavaClientTestCase extends OpenSearchRestTestCase implements OpenSearchTransportSupport {
-    private static final List<String> systemIndices = List.of(".opensearch-observability", ".opendistro_security", ".plugins-ml-config");
+    private static final List<String> systemIndices = List.of(
+        ".opensearch-observability",
+        ".opendistro_security",
+        ".plugins-ml-config",
+        ".tasks"
```

Comment:
This was causing the pipeline failure 🤦 so I have added the .tasks system index here.